src/views/blocks/tickets.php

```diff
@@ -12,32 +12,34 @@
  * @since   4.9
  * @since   4.10.8  Updated loading logic for including a renamed template.
```

Comment:
@juanfra These changes are all good, but are they all necessary for the work we're doing in V2? If not, how many of these view changes can/should be backed out to limit how much we affect the V1 tickets view?

Reply:
@sc0ttkclark Yes, this is just because we started without the v1/v2 split, and the changes were already in place and part of the PR. What I'll do from now onwards is just working on v2 templates, atomizing templates, reducing the logic inside templates, organizing into folders, and removing the changelogs on each template. A fresh start.

Reply:
Given that a fix is needed in this template for classes, we can keep the other changes